scripts/deploy.sh

```diff
@@ -171,7 +171,7 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   if [[ "${DEPLOY_TEMPEST}" == "yes" ]]; then
     # Deploy tempest
-    run_ansible os-tempest-install.yml
+    run_ansible ${BASE_DIR}/scripts/run_tempest.yml
```

Comment:
It looks like tempest gets deployed, then again. If you set ``tempest_install: no`` in the overrides above, then tempest will only be deployed is ``DEPLOY_TEMPEST == "yes"``.